Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Game.java

```diff
@@ -0,0 +1,115 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+import java.util.ArrayList;
+
+public class Game {
+	protected 	Dictionary 	dictionary;		
+	protected 	Player 		player;		  	
+	protected 	char[] 		etalon;			
+	protected	ArrayList 	<Character> 	guessedLetters;
+	protected	char[] 		openedWord;		
+	protected	ArrayList	<Character>		wrongLetters;
+	protected 	UserInterface ui;			
+	protected 	final int 	TOTAL_GUESSES=9;
```

Comment:
Good use of a constant rather than just using a literal, this makes the value easier to change, reduces mistakes, and is also makes the code more readable